src/views/InvolvementProfile/components/Membership/components/AdminCard/components/RequestsReceived/index.jsx

```diff
@@ -66,7 +62,11 @@ const RequestsReceived = ({ onAddMember }) => {
               />
 
               <ListItemSecondaryAction>
-                <Button style={redButton} onClick={() => onDeny(request.RequestID)} size="small">
+                <Button
+                  className={styles.requestReceived_deny_button}
```

Comment:
If you want a button that uses the color scheme with main as the background, and contrastText color, you can just use a color='error.'  You probably don't need it in this case, but there may be some buttons where externall CSS isn't necessary, which might simplify things